
<!-- saved from url=(0062)http://collaborations.gis.a-star.edu.sg/~cmb6/kumarv1/dfilter/ -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

<meta name="Generator" content="Microsoft Word 10 (filtered)">

<title> signal detection and estimation using next-gen sequencing data</title>

<style>
<!--
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:"Times New Roman";}
@page Section1
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;}
div.Section1
	{page:Section1;}
-->
</style>

</head>

<body lang="EN-US">
<div style="width:900px;  margin: 10px 20px 30px 50px; background:#DDDDDD;"> 
<div class="Section1">


<p class="MsoNormal"><b><span style="font-size:18.0pt">
<img src="./images/GISlogo.jpeg" alt="Logo" height="150"> 
 <img src="./images/logo.jpg" alt="Logo" height="150" width="150"> 
 
Signal detection and estimation using next-gen sequencing data </span></b></p>

<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal"> Dfilter is a generalized signal detection tool for analyzing
next-gen massively-parallel sequencing data by using ROC-AUC maximizing linear
filter. Hence it is an ideal tool for detecting peaks in tag-profile of ChIP-seq, DNase-seq, FAIRE-seq, ATAC-seq, MNase-seq, RIP-seq, CLIP-seq, ChIP-exo, Sono-seq etc.
</p>
<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal">Efilter is signal estimation tool for estimating relevant
signals using next-gen sequencing data with the help of linear filter. </p>

<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal">Both Dfilter and Efilter can take more then one
massively-parallel sequencing data and thus they provide an ideal platform for
combining patterns of different factors and markers.</p>

<p class="MsoNormal"> New versions of DFilter and Efilter are available now. With new version of 
DFilter/EFilter it is possible to use bam files as well precalculated score in bedGraph format 
<a href="./tutorial.html#bedgraph">(read more.. ) </a> . Many more features&nbsp;</p>
<p>
The extended version of DFilter can also be used to estimate read-count on given set of peaks and perform GC bias correction.
</p>
<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal"><b><span style="font-size:14.0pt">Download Dfilter and Efilter package� </span></b></p>


<p class="MsoNormal"><a href="download.htm">Download binaries or source code (click here to download)</a>   </p>
<p class="MsoNormal">Important: For running binary code, include the DFilter folder in environment variable PATH by typing </p>
<p class="MsoNormal">export PATH=path_of_DFilter_folder:$PATH </p>
<br>
	<p class="MsoNormal"><a href="download.htm">Download binaries or source code of extension of DFilter(click here to download)</a>   </p>
	<p class="MsoNormal">The extension of DFilter has many functions like normscore for read-count estimation, GC-bias correction etc </p>
	
	
<p class="MsoNormal">Matlab code: write to kumarv1@gis.a-star.edu.sg or vibhor@iiitd.ac.in</p> 

<p class="MsoNormal">&nbsp;</p>
For help and tutorial see <br>
<span style="font-size:14.0pt"><a href="./tutorial.html">Documentation (click here) for new version </a> </span> <br>
<br>

<p class="MsoNormal"><b><span style="font-size:13.0pt"><a href="./dfilter-FAQs.html"> why should I use DFilter (FAQs)�</a> </span></b></p>

<a href="https://www.dnanexus.com/"> Use Dfilter on DNAnexsus cloud computing </a><a> <br> <br> 
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">&nbsp;</p>

<p class="MsoNormal"><span style="font-size:14.0pt"><b> �Citation: </b> </span></p> 
</a><p><a>Kumar V, Muratani M, Rayan NR, Kraus P, Lufkin T, Ng HH and Prabhakar S, Uniform, optimal signal processing of mapped deep-sequencing data, Nature biotechnology, 31, 615-622 (2013) </a><a href="http://www.nature.com/nbt/journal/v31/n7/full/nbt.2596.html">  PMID: 23770639 </a>
</p>

<p> 
Publications citing the DFilter manuscript on 
<a href="http://scholar.google.com.sg/scholar?cites=7467893437717734540&amp;as_sdt=2005&amp;sciodt=0,5&amp;hl=en">
google scholar </a><a>

</a></p><a>
 <!-- comparison of DFilter to MACS, F-seq, ZINBA, QUEST, Findpeaks and SICER is done in the manuscript but users are welcome to compare against other tools also such as Sissrs, SPP, NucleR, NarrowPeaks, Homer and many other for peak-calling  in tag-profile of ChIP-seq, DNase-seq, FAIRE-seq, ATAC-seq, MNase-seq, RIP-seq, CLIP-seq, ChIP-exo, Sono-seq and other siimlar assays -->


<me="keywords" content="peak-calling peaks compare MACS F-seq ZINBA QUEST Findpeaks SICER ChIP-seq DNase-seq FAIRE-seq ATAC-seq MNase-seq RIP-seq CLIP-seq ChIP-exo Sono-seq ">
<p class="MsoNormal">&nbsp;</p>

<br>
EFilter is available with DFilter software package 
<br><br>
For using efilter: the precalculated score file
to predict expression using Histone modification for refseq genes in Human genome(hg18) and hg19 is already provided with dfilter package.

<br><br><br>
For enquiries about problems in running the programs or to get the source code
contact : kumarv1@gis.a-star.edu.sg
<br>
<p class="MsoNormal">&nbsp;</p>
</me="keywords"></a></div><a>

</a></div><a>




</a></body></html>
