
<html>
	<head>
		<title>UniPath</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	<link rel="stylesheet" href="assets/menustyles.css" />
	<script src="http://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
	<script src="script.js"></script>

	</head>
<body>
<font face="arial" color="black">

<div id='cssmenu'>
		<ul>
		
		   <a href="index.html" class ="btn">Home</a>
		   <a href='Download.html'><span>Download </span></a>
		   <a href="FAQs.html" class="btn">FAQs</a>
		   <a href="contact.html"><span>Contact</span></a>

		</ul>
		</div>

		


		<!-- Nav -->
			<nav id="menu">
				<ul class="links">
					<li><a href="index.html">Home</a></li>

					

				</ul>
			</nav>
			
		<!-- Banner -->
		<section>
				 <div style="width:1480px; height:350px; padding:70px; background-color:black; color: white">

					<header>
						<h2 class="project-name"><font color="white" face = "arial"><center>UniPath</h2></font><center>
						<h3><font color="white" face="arial"><center>Overview</h3></font><center>
						<p><font size= "4" face="arial"> UniPath provides robust statistical methods to represent every single cell using pathway and gene-set enrichment scores. It can be used with both single cell RNA-seq and single cell ATAC-seq profile with scalability for atlas scale data-sets. UniPath comes with several features like pseudo-temporal ordering using pathway scores and unconventional way of enumerating differences between two cell populations.</font></p>
					</header>
				</div>
			</section>
			
			<!-- Main -->
			<div id="main">

				<!-- Section -->
					<section class="wrapper style3">
						<div class="inner">
							<!-- 2 Columns -->
									<div class="col col1">
									  <h3 style="color:green;">Data download links</h3>
									  <p><font size=5><b>To download user must provide details as follows: User id: reggen and password  unipath@123	  </p></b>	</
									  <br> We have included precompiled pathway scores for few scATAC-seq studies using pathway annotation file having 1329 pathways which can be downloaded from the link:
									  <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/scATACseq/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/scATACseq</li></font>
									  <br><font color="black" face="arial"> Precompiled pathway scores for various studies using two pathway annotation files for scRNAseq data can be downloaded from the link: </font>
									  <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/</li></font>
									  <br><font color="black" face="arial"> Precompiled cell detection scores for various scRNAseq data can be downloaded from the link: </font>
									  <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/Cell_detection/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/Cell_detection/</li></font>
                                      <br><font color="black" face="arial"> Mandatory files required for conversion of scATAC-seq profiles to pathway scores can be downloaded from the link: </font>
									  <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Files_for_scATACseq_profile_conversion/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Files_for_scATACseq_profile_conversion/</li></font>
                                      <br><font color="black" face="arial"> Additional files: </font>
									  <br><font color="black" face="arial"> We have included tSNE coordinates for study id: GSE108097 (Single cell mouse altas dataset) in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/GSE108097/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/GSE108097/</li></font>
								      <br><font color="black" face="arial"> We have included codes and datasets used in the paper to replicate the results and figures in the following links: </font>
							          <br><font color="black" face="arial"> Codes for Figure 1 </font>

								      <br><font color="black" face="arial"> Codes and datasets used for comparison of UniPath in terms of cell type detection with AUCell,GSVA and PAGODA for Figure 1A can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1A/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1A/</li></font>
                                      <br><font color="black" face="arial"> Codes and datasets used to evaluate consistency of UniPath to give correct pathways and its comaprison with AUCell,GSVA and PAGODA for Figure 1B can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1B/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1B/</li></font>
									  <br><font color="black" face="arial"> Code and dataset used for creating barplots for Figure 1D can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1D/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1D/</li></font>

									  <br><font color="black" face="arial"> Precompiled pathway scores and code for Figure 1E can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1E/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure1/Figure1E/</li></font>

							          <br><font color="black" face="arial"> Codes for Figure 2 </font>
								      <br><font color="black" face="arial"> For Figure 2A, code and data can be accessed from the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2A/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2A/</li></font>

									  <br><font color="black" face="arial"> We have included codes and datasets used for evaluating performance of UniPath with AUCell, GSVA and PAGODA in terms of visualization and clustering for the Figure 2B and supplementary Figure S8 in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2B/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2B/</li></font>
									  
									  <br><font color="black" face="arial"> For replicating results from the Figure 2C codes and datasets can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2C/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure2/Figure2C/</li></font>

								      <br><font color="black" face="arial"> Codes for Figure 3 </font>
								      <br><font color="black" face="arial"> Codes and datasets used for running Monocle, TSCAN, cellTree and Diffusion Maps for the Figure 3A can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3A/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3A/</li></font>

                                      <br><font color="black" face="arial"> Code for pseudo temporal ordering used in case of UniPath for the Figure 3B can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3B/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3B/</li></font>

									  <br><font color="black" face="arial"> Code used in Figure 3C can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3C/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3C/</li></font>

									  <br><font color="black" face="arial"> Code and data used for replicating Figure 3D can be found in the link. This link contains three codes correponding to three figures along with a readme.txt file: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3D><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure3/Figure3D</li></font>

                                      <br><font color="black" face="arial"> Codes for Figure 4 </font>

									 <br><font color="black" face="arial">  User can use tsne coordinates directly or precompiled pathway scores for Figure 4A from the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/GSE108097/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/scRNAseq/GSE108097/</li></font>
                                      
									 <br><font color="black" face="arial"> Code and datasets used in Figure 4C can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure4/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure4/</li></font>
                                      
									  <br><font color="black" face="arial"> Codes for Figure 5 </font>
								      <br><font color="black" face="arial"> Code used for the Figure 5A can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5A/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5A/</li></font>
                                       <br><font color="black" face="arial">Code used for Figure 5B can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5B/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5B/</li></font>

								      <br><font color="black" face="arial"> Code and dataset used for plotting heatmaps of correlation of pathways in Figure 5C can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5C/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5C/</li></font>
                                      <br><font color="black" face="arial"> Code and dataset used for getting correlation values in Figure 5D can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5D/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5D/</li></font>
                                      <br><font color="black" face="arial"> Code and dataset used for replicating results in Figure 5E can be found in the link: </font> <li><a href=http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5E/><font color="blue" face="arial">http://reggen.iiitd.edu.in:1207/unipath_data/Data/Codes/Figure5/Figure5E/</li></font>


									  <br>								  
									</div>
								</div>

						