<!DOCTYPE html>
<html lang="en" dir="ltr">
 <head>
   <meta charset="utf-8">
   <title>scEChIA_Manual</title>
   <!link rel="stylesheet" type="text/css" href="scEChIA_manual_CSS.css">
 </head>
 <body>
 <div class = "image">
 <img src="scEChIA_Image.jpg" alt="">
 </div>
<h1>Introduction:</h1>

scEChIA (Single Cell Epigenome Chromatin Interaction Analysis) is a computational method that can predict chromatin interactions among distal sites with high accuracy. scEChIA can also make UCSC track on predicted interactions that may be useful for genomics studies based on chromatin interactions using the UCSC Genome Browser.

<h3>How to Install package:</h3>
<h2 class="w3-padding-12" font face="arial" color="blue"><a href="http://reggen.iiitd.edu.in:1207/scEChIA/package/scEChIA_0.1.0.tar.gz" download >Download Package</a></h2>


version.string R version 3.6.0<br>
These libraries need to be installed before the package is installed : varbvs, pracma, glassoFast, nor1mix, roxygen2.<br>
install.packages("scEChIA_0.1.0.tar.gz", repos=NULL, type="source")<br>


<h2>Library required for scEChIA package:</h2>

library(varbvs)<br>
library(pracma)<br>
library(glassoFast)<br>
library(nor1mix)<br>
library(roxygen2)<br>
library(scEChIA)<br>

<h2>Chromatin interaction prediction using differential rho:</h2>
<h2>Function_name: Interaction_Prediction_1</h2>
Interaction_Prediction_1 can predict interaction with the help of two different cell types' rho value. First we will pass HiC data from two different celltypes and take average of both HiC data to define as final rho matrix for predicting interaction from single cell data.

<h3>How to run:</h3>
setwd(system.file("data", package="scEChIA")) #Keep your input data format like example data<br>
genomic_region1 = read.table('K562_HiC_chr17.txt')<br>
genomic_region1[is.na(genomic_region1)] = 0<br>
genomic_region2 = read.table('IMR90_HiC_chr17.txt')<br>
genomic_region2[is.na(genomic_region2)] = 0<br>
gap = 25000 #Bin size 25kb (User can change it as per his choice)<br>
patternf = 17 #Chromosome number (User can change it according to chromosome number)<br>
chrNo = patternf<br>
data = read.table('GM12878_SC_chr17.txt')<br>
chrinfo = data[, 1:3] #Chromosome location (chr, start, end)<br>
chrNo = patternf<br>
startCell = 96 #start sample column<br>
endCell = 575 #end sample column<br>
chromSize = 83257441 #Size of chromosome 17 (Change it according to chromosome number. Follow heading Chromosome Size of hg19)<br>
rhomatrix = rhomatAvg(genomic_region1, genomic_region2, gap,  patternf, data, chrinfo)<br>
predicted_interaction = Interaction_Prediction_1(chrinfo, data, rhomatrix, chrNo, startCell, endCell, chromSize)<br>

<h2>Chromatin interaction prediction using constant rho:</h2>
<h2>Function_name: Interaction_Prediction_2</h2>
Interaction_Prediction_2 can predict interaction with the help of constant rho value.

<h3>How to run:</h3>
gap = 25000 #Bin size 25kb (User can change it as per his choice)<br>
patternf = 17 #Chromosome number (User can change it according to chromosome number)<br>
chrNo = patternf<br>
data = read.table('GM12878_SC_chr17.txt')<br>
chrinfo = data[, 1:3] #Chromosome location (chr, start, end)<br>
chrNo = patternf<br>
startCell = 96  #start sample<br>
endCell = 575   #end sample<br>
chromSize = 83257441 #Size of chromosome 17 (Change it according to chromosome number. Follow heading Chromosome Size of hg19)<br> 
rhoConstant = 0.01 #user can change the value of rhoConstant<br>
predicted_interaction = Interaction_Prediction_2(chrinfo, data, rhoConstant, chrNo, startCell, endCell, chromSize)<br>

<h2>How to make UCSC Track:</h2>
<h2>Function_name: ucscTrack</h2>
Function ucscTrack can create track file for UCSC Genome Browser on predicted interactions. Using addscore value it will provide perfect interaction score in decimal numbers.
<h3>How to run:</h3>
predInteraction = predicted_interaction<br>
addscore = 500 #User can increase or decrease addscore value as per his choice<br>
ucscTrack_file = ucscTrack(predInteraction, addscore)<br>
<h3>Note:</h3> 
After generating the UCSC track file, the bottom line has to be pasted on the header to be visualized on the UCSC Genome Browser.<br>
track type=interact name="User and celltype" description="User and celltype" interactDirectional=true maxHeightPixels=200:100:50 visibility=full<br>
#chrom	chromStart	chromEnd	name	score	value	exp	color	sourceChrom	sourceStart	sourceEnd	sourceName	sourceStrand	targetChrom	targetStart	targetEnd	targetName	targetStrand<br>


<h2>Chromosome Size of hg19</h2>
chr1: 248956422, chr2: 242193529, chr3: 198295559, chr4: 190214555, chr5: 181538259, chr6: 170805979, chr7: 159345973, chr8: 145138636, chr9: 138394717, chr10: 133797422, chr11: 135086622, chr12: 133275309, chr13: 114364328, chr14: 107043718, chr15: 101991189, chr16: 90338345, chr17: 83257441, chr18: 80373285, chr19: 58617616, chr20: 64444167, chr21: 46709983, chr22: 50818468, chrX: 156040895, chrY: 57227415

<h2 class="w3-padding-12" font face="arial" color="blue"><a href="./Chromatin-Interaction/Predicted-Chromatin-Interactions/">Supplementary Files</a></h2>
<div class="w3-row-padding w3-padding-64 w3-container">
  <div class="w3-content">
      <h3 class="w3-padding-32">For any issue related to execution or installation you could write email to : <a href="mailto:vibhor@iiitd.ac.in">vibhor@iiitd.ac.in</a> , 
		<a href="mailto:kumarv1@gis.a-star.edu.sg">kumarv1@gis.a-star.edu.sg</a> , <a href="mailto:neeteshp@iiitd.ac.in">neeteshp@iiitd.ac.in</a></h3>
  </div>
</div>


</body>
</html>
